src/EditorFeatures/CSharpTest/Completion/CompletionProviders/KeywordCompletionProviderTests.cs

```diff
@@ -684,5 +684,130 @@ class C
 
             await VerifyItemIsAbsentAsync(markup, "required");
         }
+
+        [Fact]
+        public async Task SuggestFileOnTypes()
+        {
+            var markup = $$"""
+                $$ class C { }
+                """;
+
+            await VerifyItemExistsAsync(markup, "file");
+        }
+
+        [Fact]
+        public async Task DoNotSuggestFileAfterFile()
+        {
+            var markup = $$"""
+                file $$
+                """;
+
+            await VerifyItemIsAbsentAsync(markup, "file");
```

Comment:
Should we verify that `class` is present here? Or is that covered by existing tests?

Reply:
Remarkably, this is not really the case for lots of different modifiers. Try taking an empty file and type `static cl$$`. You won't get a completion :)

Reply:
well thats' bad.  we should def fix that :D

Reply:
<shocked pikachu face>